kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -41,10 +60,11 @@ You will need:
   - Maven 3.5.4+ installed
 
 When using native image compilation, you will also need: 
-  - GraalVM 19.1+ installed
+  - GraalVM 19.1+ installed [prerequisites] with quarkus 1.3.0.*, GraalVM 19.3.1 minimum needed
   - Environment variable GRAALVM_HOME set accordingly
   - Note that GraalVM native image compilation typically requires other packages (glibc-devel, zlib-devel and gcc) to be installed too, please refer to GraalVM installation documentation for more details.
-
+    https://www.graalvm.org/docs/reference-manual/native-image/
```

Comment:
You can remove the `[prerequisites]`
I added it at the beginning to mention where the modification was but the line number was in fact more relevant.
Also, I would put the `GraalVM native image needs as well native-image extension [...]` before `Note that GraalVM native image compilation typically requires other packages[...]`

Reply:
fixed @radtriste